azkaban-common/src/main/java/azkaban/metrics/CommonMetrics.java

```diff
@@ -34,6 +34,8 @@
   private final MetricsManager metricsManager;
   private Meter dbConnectionMeter;
   private Meter flowFailMeter;
+  private Meter dispatchFailMeter;
+  private Meter dispatchSuccessMeter;
```

Comment:
These metrics are for executorManager, and executorManager is in azkaban-common. Unless we refactor executorManager and put it in web server, I cannot think of a way to do that properly.